api/lib/getJSON.js

```diff
@@ -1,5 +1,45 @@
+const object = require('lodash/fp/object');
+
+const reduceObj = (accumulator, currentObj) => {
+  const newAccumulator = Object.assign({}, accumulator);
+  Object.entries(currentObj).forEach(([key, value]) => {
+    const accumulatedVal = accumulator[key];
+    if (!accumulatedVal) {
+      newAccumulator[key] = value;
+    } else if (typeof accumulatedVal === 'object') {
+      newAccumulator[key] = reduceObj(accumulatedVal, value);
+    } else if (typeof accumulatedVal === 'string') {
+      newAccumulator[key] += ` ${value}`;
+    }
+  });
+  return newAccumulator;
+};
+
+const concatObjects = array => {
+  return array.reduce((accumulator, current) => reduceObj(accumulator, current), {});
+};
+
+/* eslint-disable no-param-reassign */
```

Comment:
remove

Reply:
its required. it wont pass otherwise and the logic would have to be completely rewritten by writing some other loops/arrays